examples/backends/vllm/launch/agg_multimodal_router.sh

```
@@ -1,262 +1,201 @@
-            ${VLLM_EXTRA_ARGS} &
-    # trap 'kill 0' handles cleanup
-    # Wait for this worker before starting the next one to avoid ZMQ port races.
-    wait_ready "http://127.0.0.1:${WORKER_PORT}/health" "vLLM backend $i" 900
+        --model "${MODEL}" \
+        --enable-multimodal \
+        --block-size "${BLOCK_SIZE}" \
+        --enforce-eager \
+        --gpu-memory-utilization "${GPU_MEMORY_UTILIZATION}" \
+        --max-model-len "${MAX_MODEL_LEN}" \
+        --kv-events-config "${KV_EVENTS_CONFIG}" \
+        ${GPU_MEM_ARGS} ${VLLM_EXTRA_ARGS} "${PASSTHRU_ARGS[@]}" &
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)